release-engineering/role-handbooks/branch-manager.md

```diff
@@ -198,36 +201,47 @@ Note:
+Builds against a `release-x.y` branch are implicitly a next beta. `gcbmgr` and `anago` automatically finds and increments the current build number. The command example below is to stage a build for a beta release:
 
-`./gcbmgr stage release-1.15 --build-at-head --nomock`
+`./gcbmgr stage release-1.16 --build-at-head --nomock`
 
 This being the first build from the newly created release branch, the publication of this build does not send out the typical changelog detail notification, but rather will only send a shorter message with subject line "[k8s release-1.15 branch has been created](https://groups.google.com/forum/#!topic/kubernetes-announce/Gu2JYa4BzAg)".
 
 To publish (release) the build artifacts from staging beta for example, run:
 
-`./gcbmgr release release-1.15 --buildversion=v1.15.0-alpha.3.N+commit-hash --nomock`
+`./gcbmgr release release-1.16 --buildversion=v1.16.0-alpha.3.N+commit-hash --nomock`
 
-When the new `release-X.Y` branch is created, it marks the beginning of another stage in the release process. The following tasks should take place as soon as possible:
+When the new `release-x.y` branch is created, it marks the beginning of another stage in the release process.
 
-1. Insert the patch release manager into the branch manager list in the prow configuration. For example in [`prow/plugins.yaml`](https://github.com/kubernetes/test-infra/blob/master/prow/plugins.yaml), grep for `branch_whitelist:` and add a new block for `release-X.Y`. You may need to remove the oldest `release-X.Y` block. See a pull request [example](https://github.com/kubernetes/test-infra/pull/12680) for this.
-2. [Create CI/Presubmit jobs for the new release](#Create-CIPresubmit-jobs)
-3. [Run `./branchff` approximately the day after the branch has been created]((#Branch-Fast-Forward))
+The following tasks should take place as soon as possible:
 
-## Release Candidate Stage and Release
+1. Update the [Slack branch whitelist](https://github.com/kubernetes/test-infra/blob/89e6e9743b74a646ca883c21a127773e8eb35e7d/prow/plugins.yaml#L211-L230) ([example PR](https://github.com/kubernetes/test-infra/pull/13869)):
+   - Find the current branch whitelist config ([`prow/plugins.yaml`](https://github.com/kubernetes/test-infra/blob/master/prow/plugins.yaml), grep for `branch_whitelist:`)
+   - Remove the oldest release branch block e.g., `release-1.12`
+   - Add an entry for the newest release branch e.g., `release-1.16`
+   - Add all current [Patch Release Team members](/release-managers.md#patch-release-team) to all `kubernetes/kubernetes` release branches
+   - Remove [Branch Managers](/release-managers.md#branch-managers) from any previous release branches
+   - Add all current [Branch Managers](/release-managers.md#branch-managers)
+2. [Create CI/Presubmit jobs for the new release](#create-cipresubmit-jobs)
+3. [Run `./branchff` approximately a day after the branch has been created]((#branch-fast-forward))
```

Comment:
IIUC anyone who'll need to run `branchff` needs to be in the `release-managers` GitHub team? Can we also mention that in the Onboarding section?

Reply:
We mention this in the beginning of the `Branch Content Management` section.
I'm planning to [refactor this doc a bunch and have more guidance on Release Engineering onboarding in the repo](https://github.com/kubernetes/sig-release/issues/695#issuecomment-505762265), but I'm going to punt on that for this PR.